frontend/src/pages/Create.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import BountyTitleForm from "../Components/BountyTitleForm";
+import BountyLocationForm from "../Components/BountyLocationForm";
+import BountyInfoForm from "../Components/BountyInfoForm";
+import BountyRewardForm from "../Components/BountyRewardForm";
+
+function Create() {
+    return (
+        <div>
+            <BountyTitleForm></BountyTitleForm>
```

Comment:
Like each of these forms could be 1 component and you just feed the props down from this component for each form.